### PR TITLE
Data Types practice

### PR DESCRIPTION
**Pull request by RoxanaZahan**
Created DataTypesPackage containing exercises for Enums, Strings and Maps.
Please review my code.

### CODE REVIEW

File: src/main/java/DataTypesPackage/Strings.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    public String startWord(String string, String word) {
+        String startWord = "";
+        if(string.substring(0,word.length()).equals(word) || string.substring(1,word.length()).equals(word.substring(1))) {
+            startWord = string.substring(0,word.length());
+        }
+        return startWord;
+    }
+
+    public String concatStrings(String a, String b) {
```
Comment (hosumihaidarius) [inline src/main/java/DataTypesPackage/Strings.java:13]: Please use camelCase

File: src/main/java/DataTypesPackage/MapsAndHashMaps.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package DataTypesPackage;
+
+import com.sun.xml.internal.ws.api.ha.StickyFeature;
```
Comment (hosumihaidarius) [inline src/main/java/DataTypesPackage/MapsAndHashMaps.java:3]: Please clean the useless imports

**Review by hosumihaidarius — CHANGES_REQUESTED** [review]
Review the comments and then it's fine.

LGTM, GG.

File: src/main/java/DataTypesPackage/Strings.java (hunk 1 above)
Comment (RoxanaZahan) [reply src/main/java/DataTypesPackage/Strings.java:13]: Will do.

File: src/main/java/DataTypesPackage/MapsAndHashMaps.java (hunk 2 above)
Comment (RoxanaZahan) [reply src/main/java/DataTypesPackage/MapsAndHashMaps.java:3]: Will do.

**Pull request merged**